src/main/scala/nl/knaw/dans/easy/multiDeposit/CommandLineOptions.scala

```diff
@@ -35,38 +36,42 @@ object CommandLineOptions {
     val opts = new ScallopCommandLine(conf, args)
 
     val settings = Settings(
-      appHomeDir = new File(Option(System.getProperty("app.home"))
-        .getOrElse(Properties.propOrNull("process.sip.home"))),
+      // TODO appHomeDir does a get on Option. Is there a default value for this or is failing hard
```

Comment:
@janvanmansum can you answer this one, since you commented on this before in the previous PR?